source/brailleTables.py

```diff
@@ -29,6 +30,13 @@ class TableSource(StrEnum):
 	"""The name of the scratchpad table source"""
 
 
+class TableType(StrEnum):
```

Comment:
With naming of this enum, I'd either chose for input vs output, or backtranslation vs translation. Given that the BrailleTable namedtuple has the output parameter, I'd go for output.